src/main/java/thrift/logic/commands/UpdateCommand.java

```diff
@@ -77,9 +86,18 @@ public CommandResult execute(Model model) throws CommandException {
 
         model.setTransaction(transactionToUpdate, updatedTransaction);
         model.updateFilteredTransactionList(Model.PREDICATE_SHOW_ALL_TRANSACTIONS);
+
+        if (transactionListPanel != null) {
```

Comment:
Then this block can be removed since the null checking is done right at the start of the method

Reply:
This null comparison is used instead of requireNonNull() as current JUnit tests cannot handle or test JavaFX initialization, necessitating `null` being passed as transactionListPanel variable during testing.

Reply:
Oh I see, that makes sense. Thanks for the clarification! Let's leave it as it is. 😁